mkdocs-base.yml

```diff
@@ -10,29 +10,34 @@ nav:
     - Context: ./introduction/context.md
     - Acknowledgements: ./introduction/acknowledgements.md
   - Language specification:
+    - About the grammar sections: ./language_specification/about_the_grammar_sections.md
```

Comment:
This (additional info) section "About the grammar sections" should go to the bottom of these sections, i.e., below the section "Expressions". This is the landing page for the Language specification, which should open with the "General structure".

Reply:
I initially had it there but then I thought that the reader should know how to interpret the grammar sections before reading them.

Reply:
I can understand that, but I do not want to give it that amount of prominence, so it should be more of an appendix for if people want to understand the grammar sections.

Maybe we could have a general landing page, with a how-to-read section, where this explainer could go... what do you think?

Reply:
That's a better idea, yes. An explanation of the Language specification section.

Reply:
Done. I've added a Genaral page at the beginning of the Language specification section, linking to: Tokens, Statements, Types, Expressions, and Case Sensitivity. I've moved File Extension to General, as it was just one line.